azure-spring-boot-starters/azure-sqlserver-spring-boot-starter/README.md

```diff
@@ -0,0 +1,51 @@
+## Azure SQL DB AlwaysEncrypted Spring Boot Starter
+
+[Azure SQL DB Always Encrypted](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-always-encrypted) feature allows to encrypt data in SQL and store master keys in Azure KeyVault
+
+## Sample Code
+Please refer to [sample project here](../../azure-spring-boot-samples/azure-sqlserver-spring-boot-sample).
+
+## Quick Start
+
+### Add the dependency
+
+`azure-sqlserver-spring-boot-starter` is published on Maven Central Repository.
+If you are using Maven, add the following dependency.  
+
+```xml
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-sqlserver-spring-boot-starter</artifactId>
```

Comment:
How about change the artifactId name to another name with `always encrypted` info included? e.g., azure-encryped-sqlserver-spring-boot-starter

Reply:
Missing `<version>xxx</version>` in the xml?

Reply:
The name is already  very long, I'd suggest to keep it as "azure-sqlserver-spring-boot-starter" - we may have more sql specific features to add in the future

Reply:
Better omit version, and use the latest one. Also any version change will require  update of readme..... better use the last one